packages/eslint-plugin/src/rules/expect.ts

```diff
@@ -241,6 +242,10 @@ const zeroSourceLocation: Readonly<TSESTree.SourceLocation> = {
 
 const expectTypeToken = "$ExpectType";
 
+// Based on scanner.ts
```

Comment:
for future not-us readers:
```suggestion
// Based on Typescript's scanner.ts
```